frame/bonded-finance/src/lib.rs

```diff
@@ -30,26 +30,22 @@
 	unused_extern_crates
 )]
 
-//! Bonded Finance pallet
+//! # Bonded Finance
```

Comment:
why not just use `#![doc = include_str!("../README.md")]`?

Reply:
Didn't realize this was possible, thanks. 

09c51a49f669938a6716b531fa2e2f71109747af